### DISCUSSION TITLE
Automatically split extract files by partition.

### DISCUSSION THREAD

**Discussion by SaPeresi** [genevaers/community#338] [Ideas]
This is the issue to discuss wish list items

**Comment by SaPeresi** [comment]
Automatically split extract files by partition.

**Comment by SaPeresi** [comment]
Ability to set aggregation buffer in the jcl instead of in the view.

**Comment by AOrthVector** [comment]
Automatically split extract files by partition is still done via logic text. This generic issue will be closed and a new issue created after technical discussion.
